src/components/EditableInput/EditableInput.jsx

```diff
@@ -201,7 +202,8 @@ const EditableInput = forwardRef(
         id={id}
         style={style}
         className={cx("editable-input--wrapper", className, {
-          "no-resize": autoSize
+          "no-resize": autoSize,
+          "brand-font": brandFont
```

Comment:
I guess the reason it's working is because we defined brand-font under heading? when we convert to css modules it will stop working and we will not understand why.
Maybe we will defined the css in heading in a way it will only works there and again in this component in a way it will only work here?

Reply:
as we discussed classes are injected from the a global file so the issue is not relevant per now.